packages/dso-toolkit/src/styles/components/_progress-indicator.scss

```diff
@@ -5,12 +5,20 @@
     position: relative;
     vertical-align: middle;
   }
+
+  &.dso-block {
+    align-items: center;
+    display: flex;
+    justify-content: center;
+
+    &.dso-gray {
```

Comment:
Moet dat per se een combi zijn met `.dso-block`, of zou het ook los kunnen/mogen?

Reply:
Geen kleur zonder dat het een "block" is, dat waren toch de specs? Anders zou je ook een linksuitgelijnd blok met achtergrondkleur zonder padding kunnen maken.

Reply:
dat vroeg ik me dus af. ok.